boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -23,28 +23,133 @@
             "type": "String",
             "value-key": "<output_dir>"
         },
+        {
+            "description": "FreeSurfer license file",
+            "id": "fs_license_file",
+            "name": "fs_license_file",
+            "optional": false,
+            "type": "File",
+            "value-key": "<fs_license_file>"
+        },
         {
             "command-line-flag": "--bold_task_type",
             "description": "The task label of BOLD images",
```

Comment:
Please provide more detailed description and examples - perhaps just copy past from docs. Or users will specify multiple values separated by comma, hyphen etc